lib/dialyzer/test/r9c_SUITE_data/src/inets/httpd_response.erl

```diff
@@ -317,7 +317,7 @@ transform({content_type,undefined})->
     ["Content-Type:text/plain\r\n"];
 
 transform({date,undefined})->
```

Comment:
@uabboli  Please comment on how you want to handle this test code.

Reply:
Modify the code any way you like to get rid of the warning(s).